octopusdeploy/schema_polling_subscription_id.go

```diff
@@ -0,0 +1,24 @@
+package octopusdeploy
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+func getPollingSubscriptionIDSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"id": {
+			Computed:    true,
+			Description: "The generated polling subscription ID.",
+			Type:        schema.TypeString,
+		},
+		"polling_uri": {
+			Computed:    true,
+			Description: "The URI of the polling subscription ID.",
+			Type:        schema.TypeString,
+		},
+		"dependencies": {
+			Optional:    true,
+			Type:        schema.TypeMap,
+			Description: "Optional map of dependencies that when modified will trigger a re-creation of this resource.",
+			ForceNew:    true,
+		},
```

Comment:
Using this in combination with a `replace_triggered_by` lifecycle hook on the helm release resource will allow the agent to "re-register" (It actually a re-install) so you could rotate the certificate and subscription IDs if you wanted to